app/experimenter/experiments/constants.py

```diff
@@ -187,13 +192,30 @@ class Type(models.TextChoices):
     }
 
     VALID_PUBLISH_STATUS_TRANSITIONS = {
-        PublishStatus.IDLE: (PublishStatus.REVIEW, PublishStatus.APPROVED),
+        PublishStatus.IDLE: (
+            PublishStatus.REVIEW,
+            PublishStatus.DIRTY,
+        ),
+        PublishStatus.DIRTY: (
+            PublishStatus.REVIEW,
+            PublishStatus.DIRTY,
```

Comment:
Do we need to have Dirty > Dirty here?  If so why isn't there Idle > Idle above?